modules/interactive_music/doc_classes/AudioStreamInteractive.xml

```diff
@@ -68,36 +85,53 @@
 				Return the [AudioStream] associated with a clip.
 			</description>
 		</method>
-		<method name="get_transition_fade_beats" qualifiers="const">
+		<method name="get_transition_fade_beats" qualifiers="const" deprecated="Use [method get_transition_fade_length] and [method get_transition_fade_length_unit] instead.">
 			<return type="float" />
 			<param index="0" name="from_clip" type="int" />
 			<param index="1" name="to_clip" type="int" />
 			<description>
-				Return the time (in beats) for a transition (see [method add_transition]).
+				Returns the same value as [method get_transition_fade_length].
+			</description>
+		</method>
+		<method name="get_transition_fade_length" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="from_clip" type="int" />
+			<param index="1" name="to_clip" type="int" />
+			<description>
+				Return the time for a transition (see [method add_transition_with_unit]). To get the unit of time
+				use [method get_transition_fade_length_unit].
+			</description>
+		</method>
+		<method name="get_transition_fade_length_unit" qualifiers="const">
+			<return type="int" enum="AudioStreamInteractive.FadeLengthUnit" />
+			<param index="0" name="from_clip" type="int" />
+			<param index="1" name="to_clip" type="int" />
+			<description>
+				Returns the unit of transition fade length. See [enum FadeLengthUnit] for available values.
```

Comment:
Assuming it's what this means:
```suggestion
				Returns the unit of length for the given transition fade. See [enum FadeLengthUnit] for available values.
```

Reply:
yea don't know what I was writing there :laughing: